src/main.rs

```diff
@@ -0,0 +1,61 @@
+use k2::{
```

Comment:
This file should probably be an example:
https://doc.rust-lang.org/cargo/reference/manifest.html?highlight=examples#examples

Reply:
Thanks, I didn't know about `cargo run --example`!